### DISCUSSION TITLE
Files Upload plugin missing

### DISCUSSION THREAD

**Discussion by jpkelly** [strapi/strapi#6240] [Legacy Topics]
I am building according to this guide.
https://strapi.io/documentation/3.0.0-beta.x/deployment/amazon-aws.html

The instructions say: "From the left-hand menu, click Plugins and then the cog wheel located to the right of Files Upload."
When I go to the Plugins page I do not see the Files Upload plugin.

**Comment by lauriejim** [comment]
Thank you for highlighting this issue.

According to the upload documentation it's now a config file to configure your provider.

https://strapi.io/documentation/3.0.0-beta.x/plugins/upload.html#using-a-provider

Could you please open a PR to fix the documentation. And tell that you have to configure the file instead of configure that in the interface.
I will appreciate your help on this update. Thank you.
